public/index.html

```diff
@@ -40,4 +41,6 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
   </body>
+  <script src="https://www.gstatic.com/firebasejs/8.0.1/firebase-app.js"></script>
```

Comment:
Are you sure these are necessary? Because we should be getting everything from firebaseconfig? Can @halitbatur confirm?

Reply:
these are not necessary

Reply:
removed